src/applications/claims-status/tests/e2e/fixtures/file-upload-test.txt

```diff
@@ -0,0 +1 @@
+This is a test file.
```

Comment:
id remove this since its no longer being used

Reply:
Oh strange, I must've missed committing that deletion.